packages/block-editor/src/components/block-card/style.scss

```diff
@@ -18,7 +18,7 @@
 
 .block-editor-block-card__title {
 	font-weight: 500;
-	margin-bottom: 5px;
+	margin: 0 0 5px;
```

Comment:
The margin declaration is being overridden by [this one](https://github.com/WordPress/gutenberg/blob/3c095a72f724ce8a7fe39b908796909c0a8a95ba/packages/edit-post/src/components/sidebar/style.scss#L59) that has a slightly higher specificity 😞 . Better duplicate the selector here: `.block-editor-block-card__title.block-editor-block-card__title` to make sure we get the correct margin.

Reply:
Thx for the comment, didn't knew that it's possible to duplicate selector to increase specificity.

![image](https://user-images.githubusercontent.com/1477453/76948090-41590380-690f-11ea-8f57-5681fc247a4d.png)